docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+  that is not real.
+   
+* One way to avoid the interpolation debate is to regrid both to a third
+  grid.
+   
+Regridding in METplus can be completed using the Regrid-Data-Plane tool if
+the fields will be used more than once.
+
+Regridding can also be done on the fly using the {Tool}_REGRID_TO_GRID.
+All grid-to-grid verification tools have the regridding capability in it.
+
+What is the evaluation time period? Retrospective with specific dates? Ongoing, near-real-time evaluation? Or both retrospective and realtime?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Basically, running retrospectively means that the observations/analyses are
+already available on disk and running in realtime is when the system needs
+to wait for the observations to be available on the system.
+
+In METplus, the LOOP_BY configuration can be used.
+
+LOOP_BY = VALID or REALTIME to have METplus proceed through the data based
+on Valid Time.
+
+LOOP_BY = INIT or RETRO to have METplus proceed through the data based
+on Initialization Time.
+
+How should the testing and evaluation project be broken down into METplus
+Use Cases? One large one or multiple smaller ones?
+
+* How will METplus be run? Manually? Scheduled, through cron?
+  Automated via a workflow manger (e.g. Rocoto, EC-Flow, Rose-Cylc)?
+
+  * If run manually, this can be done.
+    
+  * If scheduled through cron, a bash or csh script can be written to
+    set up environment variables to pass into METplus.
+    
+  * If automated via a workflow manager, it is recommended the user consider
+    configuring the use cases to run smaller amounts of data.
+    
+* Where will METplus be run? Local machine, project machine, HPC system,
+  in the cloud (e.g. AWS)? Serial runs or parallelized?
+  
+  * Running on linux or a project machine – identify where METplus is
+    installed by running **which run_metplus.py**; it is recommended an
+    additional user.conf or system.conf file is passed into the
+    **run_metplus.py** to direct where output should be written.
+    
+  * Running on HPC systems - check with the system admin to see if it
+    has been configured as a module and how to load netCDF and Python
+    modules.  For NOAA and NCAR HPCs systems, please refer to the
+    `Existing Builds <https://dtcenter.org/community-code/metplus/download>`_
+    pages for the desired version for instructions on how to load the METplus
+    related modules.
+    
+  * Running on Cloud (AWS) - these instructions are coming soon.
+    
+  * Running in parallel - As of MET v10.1.0 Grid-Stat can be run in parallel.
+    Please reach out via
+    `METplus Discussions <https://github.com/dtcenter/METplus/discussions>`_
+    if help is needed.
+
+Would a flowchart help to provide clarity?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Consider creating one or more flowcharts to illustrate the data flow
```

Comment:
May also want to encourage this route with "Utilizing a flowchart can assist in identifying which verification steps can be completed by which METplus tools", or something of that nature.

Reply:
Thanks!  Done.